mkdocs.yml

```
@@ -85,13 +85,17 @@ nav:
         - geps/gep-1282.md
         - geps/gep-1619.md
         - geps/gep-1897.md
+        - geps/gep-1867.md
       - Prototyping:
         - geps/gep-1709.md
       - Experimental:
         - geps/gep-1748.md
         - geps/gep-1016.md
         - geps/gep-957.md
         - geps/gep-713.md
+      - Implementable:
+        - geps/gep-1426.md
+        - geps/gep-1686.md
```

Shouldn't Implementable come before Experimental, like in the flow chart?
```suggestion
      - Implementable:
        - geps/gep-1426.md
        - geps/gep-1686.md
      - Experimental:
        - geps/gep-1748.md
        - geps/gep-1016.md
        - geps/gep-957.md
        - geps/gep-713.md
```

---

Good catch, thanks @frankbu!